alfresco-solrapi-client/alfresco-solrapi-client-ditto/src/test/java/eu/xenit/alfresco/solrapi/client/ditto/GetNodesMetadata.java

```diff
@@ -24,10 +27,16 @@ private static NodeView nodeViewMock() {
                         .setTxnId(6L)
                         .withProperty(Content.NAME, "Company Home")
                         .withProperty(Content.OWNER, "System")
+                        .withProperty(System.NODE_UUID, "company_home")
```

Comment:
is this intentionally not a valid uuid?

Reply:
Not intentionally, because it can :smirk: